### ISSUE TITLE
都是预测英文+数字  转模型前预测的结果都是英文+数字  转模型后预测的结果都是汉字  转模型用的config配置文件是训练是用的  求求问题

### ISSUE THREAD

**Issue by zzwwr** [PaddlePaddle/PaddleOCR#5627]

**Comment by zzwwr** [comment]
>
![N$Z@WKW~YRDEW%)B~RT K5J](https://user-images.githubusercontent.com/83124400/156709554-146aa203-a414-4661-b7e8-c59bb0fceb43.png)
![9 MQNHO5PAAK3H1(6H %ROO](https://user-images.githubusercontent.com/83124400/156709561-a56511cf-acda-49c6-ba2a-70f221d46247.png)

**Comment by zzwwr** [comment]
第一张是转模型之前 第二张是转模型之后

**Comment by zzwwr** [comment]
这个是配置文件内容

Global:
  use_gpu: true
  epoch_num: 1000 #训练批次X
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/model/
  save_epoch_step: 3
  # evaluation is run every 2000 iterations
  eval_batch_step: [0, 2000]
  cal_metric_during_train: True
  pretrained_model: G:\XuGuangOCR\PaddleOCR-release-2.4\en_number_mobile_v2.0_rec_slim_train/best_accuracy # 权重文件
  checkpoints:
  save_inference_dir: ./
  use_visualdl: False
  infer_img: train_data/word_10.png
  # for data or label process
  character_dict_path: ppocr/utils/EN_symbol_dict.txt
  max_text_length: 25
  infer_mode: False
  use_space_char: True #添加空格识别改为True
  save_res_path: ./output/rec/XGOCR.txt

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    learning_rate: 0.0005
  regularizer:
    name: 'L2'
    factor: 0

Architecture:
  model_type: rec
  algorithm: CRNN
  Transform:
  Backbone:
    name: MobileNetV3
    scale: 0.5
    model_name: large
  Neck:
    name: SequenceEncoder
    encoder_type: rnn
    hidden_size: 96
  Head:
    name: CTCHead
    fc_decay: 0

Loss:
  name: CTCLoss

PostProcess:
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc

Train:
  dataset:
    name: SimpleDataSet
    data_dir: G:/XuGuangOCR/PaddleOCR-release-2.4/train_data/
    label_file_list: ["G:/XuGuangOCR/PaddleOCR-release-2.4/train_data/rec_gt.txt"]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 100]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: True
    batch_size_per_card: 1
    drop_last: True
    num_workers: 1
    use_shared_memory: False

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: G:/XuGuangOCR/PaddleOCR-release-2.4/train_data/
    label_file_list: ["G:/XuGuangOCR/PaddleOCR-release-2.4/train_data/rec_gt.txt"]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 100]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1  # 单卡训练
    num_workers: 1
    use_shared_memory: False

**Comment by tink2123** [comment]
使用模型预测的时候，需要手动修改识别字典，默认使用的是中文字典。
--rec_char_dict_path="/your/dict/path"

**Comment by hahawo10** [comment]
楼主你好，我也是这个问题，请问是如何解决的呢

**Comment by 2859102097** [comment]
> 使用模型预测的时候，需要手动修改识别字典，默认使用的是中文字典。 --rec_char_dict_path="/your/dict/path"

谢谢您 很有效
